### PR TITLE
feat(index.html): improve UI added in v3

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

The tab names are changed to Japanese. In addition, the design of the "Others" page will be improved.

## How Has This Been Tested?

- The modified buttons work properly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/renderer/main/index.html [hunk 1]
```diff
@@ -756,61 +756,82 @@ <h4>apmの更新</h4>
...
+										</a>
+										<a
+											href="https://discord.gg/YEQRqnGsG2"
+											role="button"
+											class="btn btn-outline-primary"
+										>
+											<i class="bi bi-discord"></i> Discord
+										</a>
 									</div>
 								</div>
```
Comment (hal-shu-sato) [inline src/renderer/main/index.html:784]: How about this?
```html
								<div class="row mb-3">
									<p class="col-sm-6 col-form-label">
										<span class="app-name"></span>について
									</p>
									<div class="col-sm-6 btn-group-vertical">
										<button
											type="button"
											class="btn btn-outline-primary"
											id="open-about-window"
										>
											このアプリについて
										</button>

										<div class="btn-group" role="group">
											<a
												href="https://github.com/team-apm/apm"
												role="button"
												class="btn btn-outline-primary"
											>
												<i class="bi bi-github"></i> GitHub
											</a>
											<a
												href="https://discord.gg/YEQRqnGsG2"
												role="button"
												class="btn btn-outline-primary"
											>
												<i class="bi bi-discord"></i> Discord
											</a>
										</div>
									</div>
								</div>
```

**Review by hal-shu-sato — COMMENTED** [review]
I have a suggestion.

**Comment by mitosagi** [comment]
I have fixed it.

**Review by hal-shu-sato — APPROVED** [review]
It is a good improvement!
I think users will be able to use apm easily.

**Pull request merged**